doc/source/ray-air/check-ingest.rst

```diff
@@ -3,19 +3,12 @@
 Configuring Training Datasets
 =============================
 
-This page is a guide for setting up, understanding, and diagnosing data ingest problems in Ray AIR.
-
 Data ingest is the process of loading data from storage, apply preprocessing steps, and feeding the data into Trainers in AIR.
```

Comment:
Initially I found this sentence jarring because the title is "Configuring Training Datasets" but the first sentence then is about "data ingest"... could you reconcile the two?

Reply:
Rewrote it. Take a look?

Reply:
nice